api/db.ts

```diff
@@ -1,4 +1,16 @@
-//perhaps we want just regular postgres library, but pg-promise will be fine
-const pgp = require('pg-promise')(/*options*/)
-//TODO: get credentials from a file not included in git
-export default pgp('postgres://postgres:postgres@localhost:5432/va_trial')
+import { Pool } from 'pg';
+// TODO: get credentials from a file not included in git
+
+let connection = new Pool({
+  user: 'postgres',
+  host: 'localhost',
+  database: 'va_trial',
+  password: 'postgres',
```

Comment:
Is there some way to hide the password when doing a commit? I know we'll be using a config file or something in the future. I suppose you can just comment out the password before pushing commits?

Anyways, everything else looks good to me, and I don't think having a dummy password should prevent a merge.

Reply:
My authenticate branch (which was prematurely merged, you should review it just to see how it works) solves this problem